rescript/extract_seq_segments.py

```diff
@@ -16,6 +16,7 @@ def extract_seq_segments(input_sequences: DNAFASTAFormat,
                          perc_identity: float = 0.7,
                          target_coverage: float = 0.9,
                          min_seq_len: int = None,
+                         max_seq_len: int = None,
```

Comment:
I think it would be better if the parameters min_seq_len and max_seq_len where not set to None per default but rather directly to the vsearch default parameters of 32 and 5000 (if I looked correctly in the documentation). 
I think this would be more clear and a user would not have to check in the vsearch documentation for the parameters. 
Or is there a reason why you set them to None? 
```suggestion
                         min_seq_len: int = 32,
                         max_seq_len: int = 5000,
```
```
  --minseqlength INT          min seq length (clust/derep/search: 32, other:1)
  --maxseqlength INT          maximum sequence length (50000)
```

Reply:
Historically we've avoided hard setting program defaults. The main reason is that there have been cases in the past where the developer changed the default settings, and the QIIME package was not updated... leading to differences in the output processed by QIIME compared to the original tool. This requires us to be more vigilant in updating our defaults. This is why many options in several QIIIME 2 plugins state that "default program settings will be used". This way we do not have to worry about parameter changes. I'd rather like to avoid setting these. I can mention these common default settings as recommendations in the help text.